tests/unit/mathematics/linalg/Add_unittest.cc

```diff
@@ -63,9 +64,12 @@ TEST(AddMatrix, native_backend)
 	}
 	
 	linalg::add<linalg::Backend::NATIVE>(A, B, C, alpha, beta);
+	linalg::add<linalg::Backend::NATIVE>(A, B, D);
 	
-	for (int32_t i=0; i<9; i++)
+	for (int32_t i=0; i<9; i++) {
```

Comment:
Just for the sake of following our usual style, open and close braces in new lines. Here and below.